### PR TITLE
Use fetch instead of jQuery.ajax for the Password onboarding step

### PR DESCRIPTION
**Pull request by goldoraf**
I used fetch-mock for the testing part. Review welcome!

### CODE REVIEW

File: client/test/config/steps/welcome.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+'use strict';
+let assert = require('chai').assert;
+let fetchMock = require('fetch-mock');
+
+describe('Step: welcome', () => {
+	let jsdom;
```
Comment (gregorylegarec) [inline client/test/config/steps/welcome.js]: Typo : tab instead of spaces

File: client/test/config/steps/password.js [hunk 2]
```diff
@@ -1,26 +1,22 @@
 'use strict';
 let assert = require('chai').assert;
 let sinon = require('sinon');
-
+let fetchMock = require('fetch-mock');
 
 describe('Step: password', () => {
-
-    let jQuery;
+
```
Comment (gregorylegarec) [inline client/test/config/steps/password.js]: Typo: unexpected tabulation

**Comment by goldoraf** [comment]
Fixed!

**Comment by gregorylegarec** [comment]
Thanks @goldoraf !

**Comment by jsilvestre** [comment]
Congrats on your first PR!

**Pull request merged**